lib/modules/keyboardNav.js

```diff
@@ -1219,7 +1219,18 @@ function reply(selected = getSelected()) {
 	));
 }
 
-function navigateTo(newWindow, href) {
+function navigateTo(newWindow, href, leaveNP = false) {
+	if (leaveNP) {
+		// convert relative links to absolute (needed for the call to nonNpLocation below)
+		if (href.startsWith('/')) {
+			const url = new URL(location.href);
+			url.pathname = href;
+			href = url.href;
+		}
+
+		href = NoParticipation.nonNpLocation(href);
```

Comment:
Yes, that is a problem. I will try to figure out how to store the subdomain the user was on before entering np, so it can be restored in NoParticipation.nonNpLocation

Reply:
Pushed a fix. Using goMode no longer forces users onto www, unless they were in NP.